addons/project/models/project.py

```diff
@@ -699,6 +699,49 @@ def unlink(self):
         analytic_accounts_to_delete.unlink()
         return result
 
+    @api.model
+    def search(self, domain, offset=0, limit=None, order=None, count=False):
```

Comment:
Suggestion: Override ``_inherits_join_calc`` instead, so that the same behavior is applied to ``name_search``

+ Add test suite.